reference/clinic.v1.yaml

```diff
@@ -1548,6 +1548,21 @@ paths:
             schema:
               $ref: '#/components/schemas/PatientSummary'
         description: ''
+  '/v1/patients/summary/{summaryId}':
```

Comment:
This overlaps with `/v1/patients/{userId}` routes and must be changed. All other top-level endpoints are scoped either to a specific clinic or a user. Can you think of an alternative? `DELETE /v1/summaries/{summaryId}/clinics`?

Reply:
Requires a chart update, but fair enough.